content/influxdb/cloud/organizations/buckets/create-bucket.md

```diff
@@ -44,72 +53,92 @@ There are two places you can create a bucket in the UI.
 
 ## Create a bucket using the influx CLI
 
-Use the [`influx bucket create` command](/influxdb/cloud/reference/cli/influx/bucket/create)
-to create a new bucket. A bucket requires the following:
-
-- bucket name
-- organization name or ID
-- retention period (duration to keep data) in one of the following units:
-  - nanoseconds (`ns`)
-  - microseconds (`us` or `µs`)
-  - milliseconds (`ms`)
-  - seconds (`s`)
-  - minutes (`m`)
-  - hours (`h`)
-  - days (`d`)
-  - weeks (`w`)
-
-  {{% note %}}
-  The minimum retention period is **one hour**.
-  {{% /note %}}
+To create a bucket with the `influx` CLI, use the [`influx bucket create` command](/influxdb/cloud/reference/cli/influx/bucket/create)
+and specify values for the following flags:
+
+| Requirement                    | Include by |
+|:-------------------------------|:-----------|
+| Organization                   | `-o`       |
+| Bucket                         | `-n`       |
+| Retention Period Duration      | `-r`       |
```

Comment:
What are you thoughts on using a list instead of a table here?

```suggestion
- `-o`, `--org`: Organization name
- `-n`, `--name`: Bucket name
- `-r`, `--retention`: Retention period duration
```

Reply:
> What are you thoughts on using a list instead of a table here?

I prefer the list.